### ISSUE TITLE
UI will not properly save config changes when training

### ISSUE THREAD

**Issue by Hatsyyy** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#636]
Hey! Recently after numerous tests ( around 5-8 model training sessions )
I've noticed there is an issue with webui session config saving.

we all know how base template file for settings is in " configs " folder, being, say, 48k v2 one.
It's default batch size is 4 and total epochs 20000.
Now, an example of the issue.
I set my batch_size to 12 and total epochs to 1000, during the " train model " phase, the config saved to model's folder is not changed / updated with the changes - my own values.
It's as if it's being copied 1:1 from configs folder. As of now the only way to handle it is manually edit the 48k v2 up to your needs and / or edit it yourself as a copy, rename to config and move beforehand, before training, to your model folder.
I think this issue should be addressed as essentially, majority of users as of now are training using default batch_size 4 and total epochs 20k, which, if the learning rate or learning rate decay is being scaled on max epochs chosen, it's pretty bad ngl.

I gotta mention I use Kalo's notebook + v2 Mangio crepe fork of rvc, not sure how much it changes tho since Mangio's not playing around anything else but features such as crepe and so on, so I doubt it's related essentially to that fork in particular ( could be wrong tho ).
Thanks!
